dashboard/report/report_generator.py

```diff
@@ -0,0 +1,12 @@
+import jinja2
+from datetime import datetime
+
+
+def report_generator(content: dict):
```

Comment:
I'd suggest to call this just `generate_jinja_report` as `report_generator` falsly indicates more of a class rather than a function